src/components/Popup/EntryComponent.vue

```diff
@@ -131,6 +141,20 @@ export default Vue.extend({
         entry.type !== OTPType.steam
       );
     },
+    shouldShowFavicon() {
```

Comment:
Shouldn't this be part of `computed`?

Reply:
Will it be updated if we move it to computed after the setting is changed?

Reply:
Vuex stores are reactive. https://v3.vuex.vuejs.org/guide/state.html